Core.UnitTests/Serialization/SerializationEventRaiserTest.cs

```diff
@@ -78,20 +78,21 @@ public void InvokeAttributedMethod_UsesCache ()
       ClassWithDeserializationEvents instance = new ClassWithDeserializationEvents ();
       StreamingContext context = new StreamingContext();
 
-      MockRepository repository = new MockRepository();
-      SerializationEventRaiser eventRaiserMock = repository.StrictMock<SerializationEventRaiser>();
+      var eventRaiserMock = new Mock<SerializationEventRaiser> (MockBehavior.Strict);
+      eventRaiserMock.Setup (_ => _.InvokeAttributedMethod (instance, typeof (OnDeserializedAttribute), context)).Verifiable();
+      eventRaiserMock
+          .Protected()
+          .Setup<IReadOnlyCollection<MethodInfo>> ("FindDeserializationMethodsWithCache", typeof (ClassWithDeserializationEvents), typeof (OnDeserializedAttribute))
+          .Returns (new List<MethodInfo>());
 
-      eventRaiserMock.InvokeAttributedMethod (instance, typeof (OnDeserializedAttribute), context);
-      LastCall.CallOriginalMethod (OriginalCallOptions.CreateExpectation);
+      Assert.That (
```

Comment:
actually, is this assert as a whole redundant? it calls the method we've stubbed before?